provider/pkg/provider/helm_release.go

```diff
@@ -1102,6 +1105,7 @@ func (r *helmReleaseProvider) Delete(ctx context.Context, req *pulumirpc.DeleteR
 		}
 		uninstall.Timeout = time.Duration(timeout) * time.Second
 	}
+	// TODO: once https://github.com/helm/helm/pull/12109 is merged, use uninstall.RunWithContext
```

Comment:
Could you also create a GH issue against pulumi/pulumi-kubernetes to track this todo item/upstream dependency and link it back to this comment as well? Thanks!

Reply:
https://github.com/pulumi/pulumi-kubernetes/issues/2582